tyrian/js/src/main/scala/tyrian/syntax.scala

```diff
@@ -0,0 +1,23 @@
+package tyrian
+
+import cats.effect.Async
+
+object syntax:
+
+  /** Make a side effect cmd from any `F[Unit]`
+    */
+  extension [F[_]](task: F[Unit])
+    def toCmd: Cmd.SideEffect[F] =
+      Cmd.SideEffect(task)
+
+  /** Make a cmd from any `F[A]`
+    */
+  extension [F[_], A, Msg](task: F[A])
+    def toCmd: Cmd.Run[F, A, A] =
+      Cmd.Run(task)(identity)
```

Comment:
`Msg` doesn't seem to be used here :)

Reply:
Thank you. 🙂
This is why I make PR's even when I think I'm the only person who will look at them. Fresh eyes in the morning.

Reply:
Yes, at one point there was another method in here next to this one that used `Msg` like:

`def toCmd(using toMsg: A => Msg): Cmd.Run[F, A, Msg]`

But it was more confusing that useful.